cmd/cluster/sre_operators/listCmd.go

```diff
@@ -324,16 +333,19 @@ func (ctx *sreOperatorsListOptions) ListOperators(cmd *cobra.Command) ([]sreOper
 					}
 				}
 
+				currentCommit = extractCommit(currentVersion[i])
 				currentVersion[i] = extractVersion(currentVersion[i])
 			}
 
 			op := sreOperator{
-				Name:      listOfOperatorNames[i],
-				Current:   currentVersion[i],
-				Expected:  latestVersion[i],
-				Status:    operatorStatus,
-				Channel:   operatorChannel,
-				CommitURL: commitUrl,
+				Name:           listOfOperatorNames[i],
+				Current:        currentVersion[i],
+				CurrentCommit:  "(" + currentCommit + ")",
+				Expected:       latestVersion[i],
+				ExpectedCommit: "(" + expectedCommit + ")",
```

Comment:
```suggestion
				CurrentCommit:  currentCommit,
				Expected:       latestVersion[i],
				ExpectedCommit: expectedCommit,
```

I think we can leave out the parenthesis since we have an extra column.

Reply:
Thank you for the suggestion, changed